sycl/test-e2e/EnqueueNativeCommand/custom-command-level-zero.cpp

```diff
@@ -0,0 +1,135 @@
+// RUN: %{build} %level_zero_options -o %t.out
+// RUN: %{run} %t.out
+
+// REQUIRES: level_zero, level_zero_dev_kit
+
+// TODO: Reenable, see https://github.com/intel/llvm/issues/14598
+// UNSUPPORTED: windows
```

Comment:
Why doesn't this work on Windows?  I didn't see any explanation in the issue.

Reply:
I've removed this